test/document_stores/test_weaviate.py

```diff
@@ -184,13 +184,13 @@ def test_query(self, ds, documents):
         # docs = ds.query(query_text, filters={"name": ["name_1"]})
         # assert len(docs) == 1
 
-        docs = ds.query(filters={"name": ["name_0"]})
+        docs = ds.query(query=None, filters={"name": ["name_0"]})
         assert len(docs) == 3
 
-        docs = ds.query(filters={"content": [query_text.lower()]})
+        docs = ds.query(query=None, filters={"content": [query_text.lower()]})
         assert len(docs) == 3
 
-        docs = ds.query(filters={"content": ["baz"]})
+        docs = ds.query(query=None, filters={"content": ["baz"]})
```

Comment:
As you added a default value to `query` these `None` values should not be required anymore.